### ISSUE TITLE
Use API for thermal comfort

### ISSUE THREAD

**Issue by Mickain26** [lelas33/plugin_peugeotcars#14]
Hello to all,
I would like to know if it is possible to activate the thermal comfort from the API.
Thanks

**Comment by lelas33** [comment]
Yes, you can start and stop the "climatisation" => It will be "air conditioner" in summer, and "heater" in winter.
But the temperature is not ajustable from the API.
It is the same function than using the MyPeugeot Phone Application.

**Comment by Mickain26** [comment]
Thank you for this answer; but I can't find the start and stop function on the API

**Comment by Mickain26** [comment]
Indeed there is the function, I got confused I thought it was to start and stop the charge of the vehicle
